Automattic/wp-calypso
# Add new 2 year plan base constants

## Proposed Changes

* Add new 2 year jetpack plan constants

## Testing Instructions

1. Check this branch out locally or use the calypso live link to go to the pricing page
2. In your 0-sandbox.php file, add the following line
```php
define( 'USE_STORE_SANDBOX', true );
```
3. In WPCOM, comment out these lines of code: 319d3-pb/
4. Try out the following products to make sure the 2 year plan shows up correctly, can be bought, and looks good on the `/me/purchases` page in Calypso
- Complete
- Scan
- Anti-spam
- Social Basic
- Social Advanced
- VideoPress
- Boost
- AI
- Stats
- Search
5. Try out any functionality to make sure the 2 year plans work as they should
![image](https://github.com/Automattic/wp-calypso/assets/65001528/d22f2a46-3b43-494a-8420-975cbf61d286)
Renewal: 
![image](https://github.com/Automattic/wp-calypso/assets/65001528/042c651e-39b9-479c-bbc0-0292593e457a)
Cancellation flow:
![image](https://github.com/Automattic/wp-calypso/assets/65001528/b892e89c-143d-416c-a931-31156b50c97c)


## Pre-merge Checklist
